### PR TITLE
Use drawTriangle instead of drawing three lines

### PR DESCRIPTION
**Pull request by AnyOldName3**
tl;dr: a trivial change making it easier to customise debug drawing. Just swaps three `drawLine` calls for one `drawTriangle` call.

In my application, we'd like to use `glPolygonOffset` so drawing Bullet's debug wireframes over geometry with matching shapes shows the physics wireframe on top of the rendered scene instead of depth fighting. This requires drawing it as triangles with the mode set to just draw the edges instead of directly as lines, so in our `btIDebugDraw` subclass, we've made it do that for `drawTriangle`. That doesn't work, though, as Bullet doesn't use `drawTriangle` even when drawing things made of triangles, so I've changed the function responsible.

For any applications *not* providing a custom `drawTriangle` override, the new and old behaviour will be indistinguishable as the default implementation in `btIDebugDraw` just draws three lines.

### CODE REVIEW

**Comment by erwincoumans** [comment]
Sure, looks good to me. Thanks!
